sogs/db.py

```diff
@@ -144,7 +144,46 @@ def database_init():
             )
             raise
 
-    # Any future migrations go here
+    # New columns that might need to be added:
```

Comment:
i would put this new hunk in its own function to mark it as a distinct new discrete unit of migration.